qcodes_contrib_drivers/drivers/HP/HP8594E.py

```diff
@@ -0,0 +1,207 @@
+import logging
+from typing import Any, Dict
+import struct
+import numpy as np
+import datetime
+import qcodes.utils.validators as vals
+from qcodes.instrument.parameter import (
+    Parameter,
+    ParameterWithSetpoints,
+    ParamRawDataType,
+)
```

Comment:
I have been doing some refactoring of the qcodes public interface. Could you update this to import paramters from qcodes.parameters and the VisaInstrument from qcodes.instrument. 

This requires qcodes 0.35 but since I already proposed bumping this below it should be fine

Reply:
Change implemented.